src/webportal/src/app/job/job-view/fabric/JobList/Ordering.js

```diff
@@ -0,0 +1,64 @@
+import {getModified, getDuration, getStatusIndex} from './utils';
+
+export default class Ordering {
+  /**
+   * @param {"name" | "modified" | "user" | "duration" | “virtualCluster” | "status" | undefined} field
+   * @param {boolean | undefined} descending
+   */
+  constructor(field, descending=false) {
+    this.field = field;
+    this.descending = descending;
+  }
+
+  apply(jobs) {
+    const {field, descending} = this;
+    let comparator;
+    if (field == null) {
+      return jobs;
+    }
+    if (field === 'name') {
+      comparator = descending
+        ? (a, b) => (a.name < b.name ? 1 : a.name > b.name ? -1 : 0)
```

Comment:
You can use localeCompare to compare strings.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare